adapters/openrtb_util.go

```diff
@@ -30,9 +30,13 @@ func makeOpenRTBGeneric(req *pbs.PBSRequest, bidder *pbs.PBSBidder, bidderFamily
 		}
 	}
 
+	cur := make([]string, 1)
+	cur[0] = "USD"
```

Comment:
Facebook asks for it in their request, figured it was universal, but you're right. Will remove.